samples/java_springboot/45.state-management/README.md

```diff
@@ -1,88 +1,135 @@
+
+```
+az deployment sub create --name "stateManagementDeploy" --location "westus" --template-file ".\deploymentTemplates\template-with-new-rg.json" --parameters appId="<appid>" appSecret="<appsecret>" botId="<botname>" botSku=S1 newAppServicePlanName="stateManagementPlan" newWebAppName="stateManagement" groupLocation="westus" newAppServicePlanLocation="westus"
+```
 
 #### To an existing Resource Group
-`az deployment group create --resource-group "<groupname>" --template-file ".\deploymentTemplates\template-with-preexisting-rg.json" --parameters appId="<appid>" appSecret="<appsecret>" botId="<botname>" newWebAppName="stateBot" newAppServicePlanName="stateBotPlan" appServicePlanLocation="westus" --name "stateBot"`
+
+```
+az deployment group create --resource-group "<groupname>" --template-file ".\deploymentTemplates\template-with-preexisting-rg.json" --parameters appId="<appid>" appSecret="<appsecret>" botId="<botname>" newWebAppName="stateManagement" newAppServicePlanName="stateManagementPlan" appServicePlanLocation="westus" --name "stateManagement"
+```
 
 ### 5. Update app id and password
-In src/main/resources/application.properties update 
-  - `MicrosoftAppPassword` with the botsecret value
-  - `MicrosoftAppId` with the appid from the first step
+
+In src/main/resources/application.properties update
+
+- `MicrosoftAppPassword` with the botsecret value
+- `MicrosoftAppId` with the appid from the first step
 
 ### 6. Deploy the code
-- Execute `mvn clean package` 
-- Execute `mvn azure-webapp:deploy -Dgroupname="<groupname>" -Dbotname="<botname>"`
 
-If the deployment is successful, you will be able to test it via "Test in Web Chat" from the Azure Portal using the "Bot Channel Registration" for the bot.
+- Execute `mvn clean package`
+- Execute `mvn azure-webapp:deploy -Dgroupname="<groupname>" -Dbotname="<bot-app-service-name>"`
+
+If the deployment is successful, you will be able to test it via  "Test in Web Chat" from the Azure Portal using the "Bot Channel  Registration" for the bot.
 
 After the bot is deployed, you only need to execute #6 if you make changes to the bot.
 
+## Bot State
 
-## Further reading
+A key to good bot design is to track the context of a conversation, so that your bot remembers things like the answers to previous questions. Depending on what your bot is used for, you may even need to keep track of state or store information for longer than the lifetime of the conversation. A bots state is information it remembers in order to respond appropriately to incoming messages. The Bot Builder SDK provides classes for storing and retrieving state data as an object associated with a user or a conversation.
 
+## Further reading
 - [Maven Plugin for Azure App Service](https://docs.microsoft.com/en-us/java/api/overview/azure/maven/azure-webapp-maven-plugin/readme?view=azure-java-stable)
```

Comment:
This link points to a 404 - Content not found

Reply:
Thanks for that